src/Process/Pool/StrategyAbstract.php

```diff
@@ -56,6 +59,41 @@ public function setAlarmProcessTypeCode(string $alarmProcessTypeCode): StrategyI
         return $this;
     }
 
+    public function canEnvironmentSustainAdditionalProcesses(): bool
```

Comment:
This will be executed for each Strategy (Server, Root, Job, etc). That behavior should be benign, but it's not correct. I need to have the memory condition be specific to a strategy. I'll fix it but it shouldn't have bearing on public behavior.